### DISCUSSION TITLE
FinOps in CSP for Azure Gov subscriptions

### DISCUSSION THREAD

**Discussion by boomer4d** [microsoft/finops-toolkit#700] [Q&A] [answered]
Good morning!

My organization (a MSFT CSP) specializing in Azure Gov. has traditionally sold our customers CSP for Azure Government subscriptions. Those subs, for whatever reason, do not support the Azure cost management tools (aside from Advisor recommendations).

Would any portion of the FinOps solution be supported or usable in a CSP for Azure gov subscription?

This is a huge struggle for both my customers as well as myself. We have no visibility into ongoing cost, forecasts, budgets, alerts, etc. I regularly have customers that will deploy a service, say a Business Critical SQL Database, and not understand the cost implications for 30 - 45 days depending on their billing cycle.

Customer gets huge bill, gets upset, calls me wanting to know why they weren't aware that their costs were going up at a rapid pace and all I can really do is shrug my shoulders and say "I don't know, let's open a ticket with Microsoft".

Any suggestions or thoughts would be greatly appreciated, thanks!

**Comment by flanakin** [comment, marked as answer]
Unfortunately, not today. I'd be more than happy to work with you to contribute this into FinOps hubs, if you have Azure Data Factory experience and time to do the work. I don't have access to a legacy CSP account but could help you navigate what would need to happen.

**Comment by boomer4d** [comment]
Thanks Michael! I don't have any ADF experience (I'm traditionally an identity guy) but do have other folks on my team that would be willing to help out that do.

I also have access to at least one, if not more, CSP for Azure Gov subscriptions.

As far as time goes, the lack of CM tools in the CSP subs for Azure Gov is a major thorn in my side so I'll make time if you think we can make this work.

Thanks much!
